js/ui/sessionMode.js

```diff
@@ -109,6 +109,40 @@ const _modes = {
             right: ['a11y', 'keyboard', 'aggregateMenu', 'userMenu',
                     'dateMenu', 'hotCorner']
         }
+    },
+
+    'user-coding': {
```

Comment:
I think this is the way it was back in the old code - we had two different sessions depending on what image was built.